src/main/java/com/github/fukkitmc/fukkit/mixin/net/minecraft/server/MinecraftServerMixin.java

```diff
@@ -210,158 +236,159 @@ public String getLevelname(MinecraftServer server, String s) { // killing two bi
+//							// Migrate world data too.
+//							try {
+//								com.google.common.io.Files.copy(oldLevelDat, new File(new File(name), "level.dat"));
+//								org.apache.commons.io.FileUtils.copyDirectory(new File(new File(s), "data"), new File(new File(name), "data"));
+//							} catch (IOException exception) {
+//								LOGGER.warn("Unable to migrate world data.");
+//							}
+//							LOGGER.info("---- Migration of old " + worldType + " folder complete ----");
+//						} else {
+//							LOGGER.warn("Could not move folder " + oldWorld + " to " + newWorld + "!");
+//							LOGGER.info("---- Migration of old " + worldType + " folder failed ----");
+//						}
+//					} else {
+//						LOGGER.warn("Could not create path for " + newWorld + "!");
+//						LOGGER.info("---- Migration of old " + worldType + " folder failed ----");
+//					}
+//				}
+//
+//				WorldSaveHandler worldnbtstorage = new WorldSaveHandler(server.getWorldContainer(), name, (MinecraftServer) (Object) this, this.dataFixer);
+//				// world =, b0 to dimension, s1 to name, added Environment and gen
+//				worlddata = worldnbtstorage.readProperties();
+//				if (worlddata == null) {
+//					worlddata = new LevelProperties(worldsettings, name);
+//				}
+//
+//				((LevelPropertiesAccess) worlddata).checkName(name);// CraftBukkit - Migration did not rewrite the level.dat; This forces 1.8 to take the last loaded world as respawn (in this case the end)
+//				WorldGenerationProgressListener worldloadlistener = this.worldGenerationProgressListenerFactory.create(11);
+//				world = Constructors.newWorld2nd(this.getWorld(DimensionType.OVERWORLD), (MinecraftServer) (Object) this, this.workerExecutor, worldnbtstorage, DimensionType.byRawId(dimension), this.profiler, worldloadlistener, gen, World.Environment.getEnvironment(dimension));
+//			}
+//
+//
+//			this.initWorld(world, worlddata, worldsettings);
+//			this.server.getPluginManager().callEvent(new org.bukkit.event.world.WorldInitEvent(((WorldAccess) world).getBukkit()));
+//			this.worlds.put(world.getDimension().getType(), world);
+//			this.playerManager.setMainWorld(world);
+//
+//
+//			if (worlddata.getCustomBossEvents() != null) {
+//				this.getBossBarManager().fromTag(worlddata.getCustomBossEvents());
+//			}
+//		}
+//		this.setDifficulty(this.getDefaultDifficulty(), true);
+//		for (ServerWorld worldserver : this.getWorlds()) {
+//			this.loadSpawn(((ThreadedAnvilChunkStorageAccess) worldserver.getChunkManager().threadedAnvilChunkStorage).getListener(), worldserver);
+//			this.server.getPluginManager().callEvent(new org.bukkit.event.world.WorldLoadEvent(((WorldAccess) worldserver).getBukkit()));
+//		}
+//
+//		this.server.enablePlugins(org.bukkit.plugin.PluginLoadOrder.POSTWORLD);
+//		this.server.getPluginManager().callEvent(new ServerLoadEvent(ServerLoadEvent.LoadType.STARTUP));
+//		((ServerNetworkIoAccess) this.networkIo).acceptConnections();
+//		// CraftBukkit end
+//	}
 
```

Comment:
Don't know why you commented out all this code above. Also please avoid copy pasting raw bukkit patches into here, atleast try to remap them first.

Also commenting off with a `/*`... `*/` would be more efficient

Reply:
I commented it because it's a dead loop.